webservices/schemas.py

```diff
@@ -640,9 +640,24 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'],
         'exclude': ('idx', 'committee', 'recipient')
     },
 )
-
 augment_schemas(ScheduleBByRecipientIDSchema)
 
+
+# Schema/PageSchema for ScheduleBByRecipient
+ScheduleBByRecipientSchema = make_schema(
+    models.ScheduleBByRecipient,
+    fields={
+        'recipient_disbursement_percent': ma.fields.Decimal(places=2),
+    },
+    options={
+        'exclude': ('idx', 'committee', 'recipient')
+    },
```

Comment:
Good work! Only one minor suggestion:  'recipient' is not in the model,   no need to specify here.
options={
        'exclude': ('idx', 'committee', 'recipient')
    }